src/boosting/gbdt.h

```diff
@@ -495,7 +495,6 @@ class GBDT : public GBDTBase {
   /*! \brief Second order derivative of training data */
   std::vector<score_t, Common::AlignmentAllocator<score_t, kAlignedSize>> hessians_;
 #endif
-
```

Comment:
could you please remove this whitespace-only change? Would remove this entire file from the diff.

Reply:
Thanks for the reminder. Resolved via https://github.com/microsoft/LightGBM/pull/4630/commits/4c260d2f529a9840291a4311baf94a4c1053e09f.